opexebo/analysis/gridscore.py

```diff
@@ -78,58 +79,56 @@ def gridscore(aCorr, **kwargs):
 #    fieldThreshold = kwargs.get("field_threshold", default.field_threshold)
 #    minOrientation = kwargs.get("min_orientation", default.min_orientation)
     debug = kwargs.get("debug", False)
-    
-    
     # normalize aCorr in order to find contours
     aCorr = aCorr / aCorr.max()
-    cFieldRadius = int( np.floor(_findCentreRadius(aCorr, **kwargs)) )
+    centre = -0.5 + np.array(aCorr.shape)/2 # centre : also [y, x]
+    cFieldRadius = int( np.floor(_findCentreRadius(aCorr)) )
     if debug:
         print("Center radius is {}".format(cFieldRadius))
 
-    if cFieldRadius in [-1, 0, 1]:# or cFieldRadius[0] > .8*aCorr.shape[0]/2 or cFieldRadius[0] > .8*aCorr.shape[1]/2:
```

Comment:
This was based on a previous addition of yours - no longer necessary?

Reply:
This did not seem to be necessary anymore (based on > 6000 example cells from imaging data).